### ISSUE TITLE
Lyrics from previous song remain when skipping

### ISSUE THREAD

**Issue by DereC4** [th-ch/youtube-music#1041]
Sometimes when changing the song, the lyrics from the previous song will remain in the "lyrics" category. This happens when the skip button is clicked. 

Example: The provided lyrics are not from the song by the Norwegian synth-pop band A-ha.

![image](https://user-images.githubusercontent.com/53978637/221339005-bd0669f8-d0ea-4d99-b5a4-54ab39c884c3.png)

However, the song does have lyrics when it was loaded up without any prior song.

![image](https://user-images.githubusercontent.com/53978637/221339095-e41b94c3-6e3c-4a8e-bce9-e86a5823fc66.png)

**Comment by Zo-Bro-23** [comment]
I've noticed this too, but not exactly sure when I've noticed it. Perhaps when I've skipped a song, like you mentioned, but I'm not a 100% sure. I've definitely faced the same issue though.

**Comment by Zo-Bro-23** [comment]
Okay I just had this issue, though it had nothing to with skipping. The lyrics for Thalatherichavar turned out to be the lyrics for Thee Minnal. When I switched to the "Up Next" tab and switched back to the "Lyrics" tab, the lyrics changed, but it was now the lyrics for Shape of You! The crazy thing is that I have not listened to either of these songs in a _long_ time, and they were not in my queue. Would really appreciate some clarity to this apparently random phenomenon :)

**Comment by Araxeus** [comment]
I think it will be fixed by changing:

https://github.com/th-ch/youtube-music/blob/8bfbbca04406e8aafeb9006c1a27fd98a25a897b/plugins/lyrics-genius/back.js#L34-L35

to 

```js
 url = info.response.sections.filter((section) => section.type === "top_hit")[0] 
 	.hits[0].result.url;
```

**Comment by DereC4** [comment]
Will try

**Comment by Araxeus** [comment]
I've opened a PR which should fix this (#1052), could you test it?
